### ISSUE TITLE
Use Travis-CI to run PHPUnit tests?

### ISSUE THREAD

**Issue by mjordan** [MarcusBarnes/mik#272]
Now that we're adding more tests, should we be configuring Travis to run them on master? Might be nice to have in place for the 1.0 milestone.

We could also test various versions of PHP (as in #271). We probably want to postpone adding PHP style checks for now :nerd_face:

**Comment by MarcusBarnes** [comment]
Agreed.  It's good to get started on this and add more checks as get past the 1.0 milestone.  Thank you for the suggestion.

**Comment by mjordan** [comment]
@MarcusBarnes I can probably find some time tomorrow to start this if you want.

**Comment by mjordan** [comment]
Thee following .travis.yml file works:

```
language: php
php:
  - 5.5
  - 5.6
  - 7.0
branches:
  only:
    master
before_install:
  - travis_retry composer self-update
  - composer install
script: phpunit --exclude-group FITS --bootstrap vendor/autoload.php tests
```

Adding this markdown immediately after the `# MIK, the Move to Islandora Kit.` should render the build badge:

```
[![Build Status](https://travis-ci.org/MarcusBarnes/mik.png?branch=master)](https://travis-ci.org/MarcusBarnes/mik)
```
@MarcusBarnes do you want me to add the travis file and the badge markdown to master, and then you can configure Travis on this repo? That should do it!

**Comment by MarcusBarnes** [comment]
@mjordan Sounds good.  Thank you.

**Comment by mjordan** [comment]
@MarcusBarnes once you've set up travis to run on the MIK repo, and then merged the issue-272 branch in, the tests should start running against the master branch (and the build status badge should show green).

**Comment by MarcusBarnes** [comment]
@mjordan Thank you.  I've set up https://travis-ci.org to run against MIK.  It uses the ``.travis.yml`` file contained in pull-request https://github.com/MarcusBarnes/mik/pull/273 as expected.  Closed with commit f445e10a9d1da30b8852f04bee77d64564bf98c9.

**Comment by mjordan** [comment]
:+1:
